### PR TITLE
[Display_01] Built DisplayGpsManagerImplmentation and Tests

### PR DESCRIPTION
**Pull request by atpoynter**
Working on #67

### CODE REVIEW

File: src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package edu.oakland.test.display01;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.Satellite;
+import edu.oakland.helper.display01.SatelliteSignalCheckRequest;
+import edu.oakland.production.display01.DisplayGpsManager;
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java:8]:
```suggestion

import edu.oakland.production.display01.DisplayGpsManager;
```

File: src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  @Test
+  @DisplayName("Indication that GPS signal strengths are received.")
+  void checkGpsSignalStrength() {
+    Satellite sat = new Satellite("word", 3);
+    DisplayGpsManager man = new DisplayGpsManagerImplementation(
+        new MiddlewareGisInterfaceStub()
+    );
+
+    SatelliteSignalCheckRequest check = new SatelliteSignalCheckRequest()
+    assertEquals("word", man.receiveGpsSignalStrength(sat));
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java]: Here is the last part of what we were working on before the stream cut off:
```suggestion
    SatelliteSignalCheckRequest check = man.passGpsSignalStrength(sat);
    assertEquals("test", check.getSatelliteName());
    assertEquals("check", check.getCheckType());
```

File: src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  }
+
+  @Test
+  @DisplayName("Indication that GPS signal strengths are received.")
+  void recheckGpsSignalStrength() {
+    Satellite sat = new Satellite("word", 3);
+    DisplayGpsManager man = new DisplayGpsManagerImplementation(
+        new MiddlewareGisInterfaceStub()
+    );
+    assertEquals("word", man.receiveGpsSignalStrength(sat));
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java]: Here is the last part of what we were working on before the stream cut off:
```suggestion
    SatelliteSignalCheckRequest check = man.passGpsSignalStrength(sat);
    assertEquals("word", check.getSatelliteName());
    assertEquals("recheck", check.getCheckType());
```

File: src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  }
+
+  @Test
+  @DisplayName("Indication that GPS signal strengths are received.")
+  void checkGpsSignalStrength() {
+    Satellite sat = new Satellite("word", 3);
+    DisplayGpsManager man = new DisplayGpsManagerImplementation(
+        new MiddlewareGisInterfaceStub()
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java]: Here is the last part of what we were working on before the stream cut out:
```suggestion
        new MiddlewareGisInterfaceStub("test")
```

**Review by andrewdimmer — CHANGES_REQUESTED** [review]
I believe this is the last of the code we need to add to be code complete for Display_01. Give it a look through, and if you agree with it go ahead and commit it. You can commit all of the suggestions as a batch at https://github.com/andrewdimmer/csi-3370-software-project/pull/135/files. Provided this passes checks, we should be good to merge.

File: src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
+package edu.oakland.test.display01;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.Satellite;
+import edu.oakland.helper.display01.SatelliteSignalCheckRequest;
+
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java]: To many spaces...
```suggestion
```

File: src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  @Test
+  @DisplayName("Indication that GPS signal strengths are received.")
+  void checkGpsSignalStrength() {
+    Satellite sat = new Satellite("word", 3);
+    DisplayGpsManager man = new DisplayGpsManagerImplementation(
+        new MiddlewareGisInterfaceStub("test")
+    );
+
+    SatelliteSignalCheckRequest check = new SatelliteSignalCheckRequest();
+    assertEquals("word", man.receiveGpsSignalStrength(sat));
```
Comment (andrewdimmer) [inline src/test/java/edu/oakland/test/display01/DisplayGpsManagerImplementationTests.java]:
```suggestion
    SatelliteSignalCheckRequest check = man.passGpsSignalStrength(sat);
    assertEquals("test", check.getSatelliteName());
    assertEquals("check", check.getCheckType());
```

**Review by EC01EC — APPROVED** [review]
if the github bots and Andrew are happy then I'm happy

**Review by andrewdimmer — APPROVED** [review]
Final coverage checks pass! You're set to merge, and Display_01 is officially code complete

**Pull request merged**